modules/lightning_features/lightning_media/src/Plugin/EmbedType/MediaLibrary.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\lightning_media\Plugin\EmbedType\MediaLibrary.
+ */
+
+namespace Drupal\lightning_media\Plugin\EmbedType;
+
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\StringTranslation\TranslationInterface;
+use Drupal\embed\EmbedType\EmbedTypeBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @EmbedType(
+ *   id = "media_library",
+ *   label = @Translation("Media Library")
+ * )
+ */
+class MediaLibrary extends EmbedTypeBase implements ContainerFactoryPluginInterface {
```

Comment:
I'm curious why this need to define a new embed type when you're basically embedding entities (which should be supported by Entity Embed, and adding a button to embed media_entity entities)?